### PR TITLE
feat: add inherited group member api

### PR DESCRIPTION
**Pull request by nanjingfm**
Add the inherited group member API in this PR.

https://docs.gitlab.com/ee/api/members.html#get-a-member-of-a-group-or-project-including-inherited-and-invited-members

### CODE REVIEW

File: group_members.go [hunk 1]
```diff
@@ -158,6 +158,31 @@ func (s *GroupMembersService) GetGroupMember(gid interface{}, user int, options
 	return gm, resp, nil
 }
 
+// GetInheritedGroupMember get a member of a group or project, including inherited and invited members
```
Comment (svanharmelen) [inline group_members.go:161]:
```suggestion
// GetInheritedGroupMember get a member of a group or project, including
// inherited and invited members
```

File: group_members_test.go [hunk 2]
```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/stretchr/testify/assert"
```
Comment (svanharmelen) [inline group_members_test.go:28]:
```suggestion
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
```

File: group_members_test.go [hunk 3]
```diff
@@ -346,3 +348,60 @@ func TestGetGroupMemberCustomRole(t *testing.T) {
...
 }
+
+func TestGetGroupMemberAll(t *testing.T) {
+	mux, client := setup(t)
+
+	path := fmt.Sprintf("/%sgroups/1/members/all/2", apiVersionPath)
+	mux.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+
+		// This is pulled straight from a `/group/<group_id>/members/all/<user_id>` call, then obfuscated.
```
Comment (svanharmelen) [inline group_members_test.go:359]:
```suggestion
```

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Few minor suggestions, rest looks good 👍🏻

**Comment by an0rak-dev** [comment]
Hi 👋🏻 
Thanks for your work ! I'm quite interested with this endpoint ! Do you have any news about this ? 👀

**Review by svanharmelen — APPROVED** [review]
LGTM! Thanks @nanjingfm 👍🏻

**Pull request merged**